### PR TITLE
Todo/week 3

### PR DESCRIPTION
**Pull request by DCHRKV**
SwiftLint installed with default settings. Most warnings were fixed. MatchedView deleted, since it was a demo for testing, it was replaced by CourseItem. CourseItem added to ScrollView in HomeView. It can be opened to CourseView.

### CODE REVIEW

**Comment by DCHRKV** [comment]
Screencast link:
[https://youtu.be/_NZ0CDy6b8A](https://youtu.be/_NZ0CDy6b8A)

File: Courses/Views/Components/CourseItem.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//  CourseItem.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 20.10.2022.
+//
+
+import SwiftUI
+
+struct CourseItem: View {
+    var namespace: Namespace.ID
```
Comment (den-ree) [inline Courses/Views/Components/CourseItem.swift]: @DCHRKV don't forget to set correct access

**Review by den-ree — APPROVED** [review]
@DCHRKV looks good, only review `private var`

File: Courses/Views/Screens/CourseView.swift [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        ZStack {
+            ScrollView {
+                cover
+                content
+                    .offset(y: 100)
+                    .padding(.bottom, 200)
+            }
+            .background(Color("Background"))
+            .ignoresSafeArea()
+            button
```
Comment (sghourab) [inline Courses/Views/Screens/CourseView.swift]: I feel button name should be more descriptive. I have the same in my code following the course instructor, but seeing it here would have made more sense it it said something like dismissButton.

**Review by sghourab — COMMENTED** [review]
Hey Dmytro, looks good to me

**Review by Forkipe — APPROVED** [review]

File: Courses/Views/Screens/CourseView.swift (hunk 2 above)
Comment (DCHRKV) [reply Courses/Views/Screens/CourseView.swift]: @sghourab , agree, fixed

File: Courses/Views/Components/CourseItem.swift (hunk 1 above)
Comment (DCHRKV) [reply Courses/Views/Components/CourseItem.swift]: @Den-Ree , I can't change it to private because I need to initialize it in preview

**Comment by DCHRKV** [comment]
*Fix according to review comments

**Review by sghourab — APPROVED** [review]

**Pull request merged**